src/components/carousel/carousel.scss

```diff
@@ -202,3 +202,7 @@ $cubic-bezier: .34,1.10,.69,1.01;
   opacity: 0;
   transition: all $slide-animation-length cubic-bezier($cubic-bezier);
 }
+
+.css-transition-group {
```

Comment:
`.carousel-transition-group-full-height` to avoid a collision.

Reply:
Oks

Reply:
I would go with a class name `carbon-carousel__transition` please, so it is consistent with the rest of the component